apps/api/src/domain/models/shared/common-commands/grant-user-read-access/grant-resource-read-access-to-user.command.ts

```diff
@@ -0,0 +1,18 @@
+import { Command, ICommand } from '@coscrad/commands';
+import { CompositeIdentifier, UUID } from '@coscrad/data-types';
+import { AggregateId, isAggregateId } from '../../../../types/AggregateId';
+import { ResourceCompositeIdentifier } from '../../../../types/ResourceCompositeIdentifier';
+import { ResourceType } from '../../../../types/ResourceType';
+
+@Command({
+    type: 'GRANT_RESOURCE_READ_ACCESS_TO_USER',
```

Comment:
We need to expose these in `availableCommands` on the individual resource classes.

Reply:
[here](https://www.pivotaltracker.com/story/show/182787868) is the story for that.